src/main/scala/com/amazon/deequ/analyzers/StateProvider.scala

```diff
@@ -224,15 +223,9 @@ case class HdfsStateProvider(
       identifier: String)
     : Unit = {
 
-    val saveMode = if (allowOverwrite) {
```

Comment:
please rebase to the latest mainline

Reply:
I believe that I have rebased to the latest master, but feel free to check if the rebase is successful or not.